### PR TITLE
Jest Test Part1

### PR DESCRIPTION
**Pull request by RolandM99**
- Use jest framework for testing.
- Mock a storage object to "imitate" localStorage operations.
- Mock HTML to test if add/delete functions add or remove exactly one <li> element to/from the list in the DOM.
- Make sure you group your tests using the description() method.

### CODE REVIEW

File: src/functionality.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  task: 'Graduate from Microverse',
+  completed: false,
+  index: 1,
+};
+
+Tasks.myTasksView(elementToAdd, createList);
+
+test("Add exactly one 'li' element in the list", () => {
+  expect(createList.childElementCount).toBe(1);
+});
```
Comment (henatan99) [inline src/functionality.test.js:21]: - [ ] Very nice work and appropriate use of mock functions. But it is the project's requirement that tests are grouped using the description() method. kindly make changes here. :pray:

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1. First clone the project with the command git clone https://github.com/RolandM99/To-Do-List.git
+2. Then run the command npm install in your terminal to open the project in your browser
+3. [optional] You can even fork the project
+
```
Comment (henatan99) [inline README.md:28]: - [ ] Kindly add scripts for starting and testing the app like `npm start` and `npm run test`. It is relevant info for this milestone. :pray:

File: src/functionality.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    expect(Tasks.getFromLocalStore()[0].task).toBe('Graduate from Microverse');
+  });
+  test('Remove element from the DOM and from LocalStorage', () => {
+    const target = document.getElementById('1 trash');
+    const childcount = createList.childElementCount;
+    Tasks.deleteTask(target, 0);
+    expect(createList.childElementCount).toBeLessThan(childcount);
+  });
+  test('Remove element from DOM and from LocalStorage', () => {
+    expect(Tasks.getFromLocalStore()).toBeDefined();
```
Comment (henatan99) [inline src/functionality.test.js:35]: - [ ] This test doesn't address the test scope 'Remove element from DOM and from LocalStorage'. Kindly make changes or try to justify here how the test and the written scope match. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @RolandM99 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
Hi @RolandM99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**